apps/andi/test/integration/andi_web/live/ingestion_live_view/metadata/metadata_form_test.exs

```diff
@@ -67,21 +67,29 @@ defmodule AndiWeb.IngestionLiveView.MetadataFormTest do
         "name" => new_name
       }
 
-      metadata_view = find_live_child(view, "ingestion_metadata_form_editor")
-      render_change(metadata_view, "validate", %{"form_data" => form_data})
-      render_change(view, "save")
+      view
+      |> form("#ingestion_metadata_form", form_data: form_data)
+      |> render_change()
 
       html = render(view)
       assert get_value(html, "#ingestion_metadata_form_name") == new_name
     end
 
-    test "dataset name field defaults to it's existing association", %{
+    test "name field shows an error if blank", %{
```

Comment:
Misleading diffs here. I've just added tests that cover error cases on an integration level.